### ISSUE TITLE
A few suggestions for wider compatibility

### ISSUE THREAD

**Issue by LoganFraser** [sb-ocr/diy-spacemouse#8]
There are many versions of the pi tiny sized rp2040 board but most don't have the i2c socket on them.  For future revisions, could you keep the i2c pins clear (4 and 5) and put the buttons on different inputs?  That would make it easier to connect to more versions.  Of course, I can just edit the pin file, and I'll have to for my seeed xiao rp2040 anyway to move the i2c.  

A 3rd button could also be easily added (and I did).  Let me know if you're interested in the file, though I'm sure you can just as easily copy the existing button setup and rotate it 90 degrees to drop in a 3rd button.  
![image](https://github.com/sb-ocr/diy-spacemouse/assets/33788254/315f9ec1-e3ca-478f-bbd9-3bc16eebb79a)

for printability, having the shaft of the knob tapered all the way to the mounting plate means no supports needed. 
![image](https://github.com/sb-ocr/diy-spacemouse/assets/33788254/924a138c-3b71-4f43-9cb2-7d73022f614e)

Same for the bottom of the case, a chamfer first and then fillet on the top half means no overhang over 45 degrees.  

Looking forward to the next version!  Thanks for sharing.  :)

**Comment by sb-ocr** [comment]
Absolutely, there are some improvements that can be made for better printability, as you pointed out. I will definitely keep that in mind for the next iteration.

Regarding the buttons, I was thinking maybe the next one could have mini clusters of 2 or 3 on each side, leaving the "palm" side where you currently placed the third button clear, just because you rest your hand on it.

Thank you for the suggestions!

**Comment by LoganFraser** [comment]
Tried out the 2 buttons per side thing.  I don't think there's room for 3, it starts getting really difficult into the rounded corners.  I had to move the mounting screws for the button plate 90 degrees.
![image](https://github.com/sb-ocr/diy-spacemouse/assets/33788254/c204a566-5d70-4b79-a7a3-39378fa95a28)

I also added a wire path for the sensor down through the middle of the shaft.  
![image](https://github.com/sb-ocr/diy-spacemouse/assets/33788254/0f2a20eb-baec-4354-aca0-27d26b1aaa02)
I should have the sensor board in a few days, it's the only piece I'm still missing.  Thanks again.
